configs/components/rubygem-bindata.rb

```diff
@@ -1,6 +1,6 @@
 component 'rubygem-bindata' do |pkg, settings, platform|
-  pkg.version '2.4.4'
-  pkg.md5sum '132d77d64bedd315acff3b28b1129877'
+  pkg.version '2.4.7'
+  pkg.md5sum '516d8371db693d375dcdc57e862aa930'
```

Comment:
md5sum mismatch. Should be `5803f4525dc4e2687b97aa37f0602a08`.

Reply:
Dang good catch!